rest-service/manager_rest/resource_manager.py

```diff
@@ -1398,6 +1398,14 @@ def get_deployment_parents_from_labels(labels):
                 parents.append(value)
         return parents
 
+    @staticmethod
+    def get_deployment_object_type_from_labels(labels):
+        types = set()
+        for label, value in labels:
+            if label == 'csys-obj-type' and value:
+                types.add(value.lower())
```

Comment:
Why happens if the user provides both `environment` and `service` as different values?

Reply:
Well, the most important  value for us is `environment` that way we can tell if this is environment or not e otherwise it will be service

Reply:
Don't we want to throw an error in such a case? 
Let's say the user updates the deployments labels to be 
```
{csys-obj-type: environment}, 
{csys-obj-type: service}
```
What should we do? I think that because it is ambiguous, we should throw an error.

Reply:
If you remember the jira we raised regarding the restrictions for label values. Ofer does not want to add such limitation to raise error to users and on this case if the deployment originally was service and user added two of these the it will be converted to environment `not deployment.is_environment and 'environment' in types`  and if it was originally `environment` and user added two values then it will not be changed since its already `environment`

Reply:
The point is that we do not want to add more complexity to the values related to the labels values

Reply:
I see. I don't think this the right behavior, but if this is the wanted behavior it's a different thing.

Reply:
I agree, but for now we are going to stick with this as we only worried about added/removed labels so that we can update the counters